src/rules/declaration-no-important/README.md

```diff
@@ -0,0 +1,14 @@
+# declaration-no-important
+
+Disallow the use of `!important` within declarations.
+
+```css
```

Comment:
Let's make it clear when we have "Bad" or "Good" examples, as for instance they do here https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md or here http://jscs.info/rules.html